codechain/keystore/HDStorageJson.py

```diff
@@ -0,0 +1,49 @@
+import binascii
+import os
+import uuid
+
+from Crypto.Cipher import AES
+
+from ..crypto import blake256
+from ..crypto import get_public_from_private
+from .Errors import ErrorCode
+from .Errors import KeystoreError
+from .keys import key_from_public_key
+from .KeyType import KeyType
+from .Pbkdf2 import pbkdf2
+from .Types import SecretStorage
+
+
+def encode(seed: str, passphrase: str, meta: str):
+    seed_hash = blake256(seed)
+    salt = os.urandom(32)
+    iv = os.urandom(16)
+
+    kdf = "pbkdf2"
+    kdf_params = {
+        "dklen": 32,
+        "salt": binascii.hexlify(salt).decode("ascii"),
+        "c": 262144,
+        "prf": "hmac-sha256",
+    }
+    derived_key = pbkdf2(
+        passphrase, salt, kdf_params.get("c"), kdf_params.get("dklen"), "sha256"
```

Comment:
I'll make another PR after this PR gets merged. 
Please prefer `[]` than `.get()`. `.get()` returns `None` silently without `KeyError`. Especially in dynamic typed language such as Python, it is prone to a bug by typo.

Reply:
Good point! I fixed them.